### DISCUSSION TITLE
PCB Files

### DISCUSSION THREAD

**Discussion by jarno9981** [micro-nova/AmpliPi#104] [Q&A] [answered]
Could you publish the PCB files, not in pdf schematics, but the actual files? I would like to build my own board for compute module 3.

**Comment by mabowers** [comment, marked as answer]
Hello @jarno9981, and thanks for your interest in AmpliPi!

AmpliPi is a radically open product. Unlike nearly all other home audio products, the AmpliPi software, firmware, and schematics are all open source! 

We provide schematics so that users are able to fully understand, maintain, and modify their AmpliPi units. However, please note that continued development of this project is funded by sales of AmpliPi hardware. Due to this reality, there are no plans for us to offer the PCB layout and other manufacturing files, as this would be detrimental to the long-term health of the project.

The AmpliPi hardware is very competitively priced compared to the current marketplace of multi-zone audio amplifiers (none of which are open source, by the way.) We also plan on offering individual system boards for sale, for users who want to integrate AmpliPi with their own hardware.
